spire/config/scripts/create-spire-mysql-server-registration-entry.sh

```diff
@@ -11,5 +11,7 @@ kubectl exec -n spire spire-server-0 -- \
     /opt/spire/bin/spire-server entry create \
     -spiffeID spiffe://example.org/mysql/server \
     -parentID spiffe://example.org/ns/spire/sa/spire-agent \
+    -x509SVIDTTL 120 \
+    -hint mysql-server \
     -selector k8s:ns:mysql \
-    -selector k8s:pod-label:app:mysql-server
\ No newline at end of file
+    -selector k8s:pod-label:app:mysql-server
```

Comment:
can we also add `-dns mysql.mysql.svc.cluster.local` here?

Reply:
Oh yeah good catch, adding